src/coreclr/src/vm/processdiagnosticsprotocolhelper.cpp

```diff
@@ -0,0 +1,210 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "common.h"
+#include "fastserializer.h"
+#include "processdiagnosticsprotocolhelper.h"
+#include "eventpipeeventsource.h"
+#include "diagnosticsprotocol.h"
+
+#ifdef FEATURE_PERFTRACING
+
+static bool IsNullOrWhiteSpace(LPCWSTR value)
```

Comment:
Where is this function used?

Reply:
Looks like it came along when I pulled in the structure fo the file from one of the other protocol helpers.  I'll remove it.